python/kserve/kserve/protocol/dataplane.py

```diff
@@ -249,13 +262,16 @@ def decode_cloudevent(self, body) -> Tuple[Union[Dict, InferRequest], Dict]:
             attributes = body._get_attributes()
             decoded_body = body.get_data()
             try:
-                decoded_body = orjson.loads(decoded_body.decode('UTF-8'))
+                decoded_body = orjson.loads(decoded_body.decode("UTF-8"))
             except (orjson.JSONDecodeError, UnicodeDecodeError) as e:
                 # If decoding or parsing failed, check if it was supposed to be JSON UTF-8
-                if "content-type" in body._attributes and \
-                        (body._attributes["content-type"] == "application/cloudevents+json" or
-                         body._attributes["content-type"] == "application/json"):
-                    raise InvalidInput(f"Failed to decode or parse binary json cloudevent: {e}")
+                if "content-type" in body._attributes and (
+                    body._attributes["content-type"] == "application/cloudevents+json"
+                    or body._attributes["content-type"] == "application/json"
```

Comment:
Use `JSON_HEADERS`?

Reply:
It'll be harder to read since `JSON_HEADERS` is just a list, so it'll just be `JSON_HEADERS[0]`, `JSON_HEADERS[1]`.